### PR TITLE
[docker] Add ubuntu16.04/18.04 devel Dockerfile

### PR DESCRIPTION
**Pull request by JeonChangMin**
Adding docker container for nnstreamer runtime and development (meson/ninja build) in ubuntu 16.04/18.04.

I added my forked repository's Dockerfile to [Docker Hub](https://hub.docker.com/repository/docker/wisechang1/nnstreamer), and build was successful.
If this PR is merged, it may be useful to create a Docker Hub nnstreamer account and add a nnstreamer/nnstreamer repository to the Docker Hub.

For nnstreamer project, docker can an easiest way for setup. (Recent days, Tensorflow recommend to use docker container, too)
Is there a better location for the `docker/` directory?

Resolves: #2657 

**Self evaluation:**
1. Build test: [ ]Passed [ ]Failed [*]Skipped
2. Run test: [ ]Passed [ ]Failed [*]Skipped

### CODE REVIEW

**Review by myungjoo — COMMENTED** [review]
Looks great in overall.

Please move to tools/docker/*.

To reduce the subdirectory depth, you may organize:
```
/tools/docker/ubuntu16.04-devel/dockerfile
/tools/docker/ubuntu16.04-run/dockerfile
/tools/docker/ubuntu18.04-devel/dockerfile
/tools/docker/ubuntu18.04-run/dockerfile
```

or

```
/tools/docker/ubuntu/dockerfile.16.04-devel
/tools/docker/ubuntu/dockerfile.16.04-run
/tools/docker/ubuntu/dockerfile.18.04-devel
/tools/docker/ubuntu/dockerfile.18.04-run
```
(users may build with ```-f <filename>``` option for such filenames)

**Review by dongju-chae — APPROVED** [review]
LGTM

File: docker/devel/ubuntu16.04/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    gstreamer1.0-x \
+    gstreamer1.0-alsa \
+    gstreamer1.0-pulseaudio \
+    libgstreamer-plugins-base1.0-dev \
+    libgtest-dev
+
+WORKDIR /opt/nnstreamer/
+RUN meson build
+RUN ninja -C build
+
```
Comment (wooksong) [inline docker/devel/ubuntu16.04/Dockerfile:35]:
```suggestion
RUN ninja -C build install 
```

Then, you do not need the following lines.

**Comment by myungjoo** [comment]
#2796 is a superset of this. Closing.

**Pull request closed without merging**